fawkes/utils/utils.py

```diff
@@ -131,3 +131,16 @@ def fetch_channel_config(app_config, channel_type):
             return review_channel
     return None
 
+def write_query_results(response, write_file, format):
+    if format == "json":
```

Comment:
Pls don't hard-code these strings, these are available in the constants.py folder